src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -222,72 +222,72 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
 
 tile.openblocks.path.name=Pfad
 tile.openblocks.path.description=Der Pfad macht überhaupt nichts. Er sieht einfach nur gut aus!
 
-tile.openblocks.item_dropper.name=Itemspender
-tile.openblocks.item_dropper.description=Der Itemspender ist dem Vanilla-Spender ähnlich, außer dass er Items direkt in einer geraden Linie nach unten wirft.
+tile.openblocks.item_dropper.name=Sachenspender
+tile.openblocks.item_dropper.description=Der Sachenspender ist dem Vanilla-Spender ähnlich, außer dass er Gegenstände direkt in einer geraden Linie nach unten wirft.
 
 tile.openblocks.big_button.name=Großer Knopf
-tile.openblocks.big_button.description=Der große Knopf ist ein Knopf, der an der Wand platziert werden kann. Wenn du auf den Knopf mit einer leeren Hand Shift-klickst, wird ein Inventar mit einem einzelnen Slot erscheinen. Du kannst Items in diesen Slot geben, um die Zeit, die der Knopf gedrückt ist, zu verlängern - ein Item erhöht die Dauer um einen Tick.
+tile.openblocks.big_button.description=Der große Knopf ist ein Knopf, der an der Wand platziert werden kann. Wenn du auf den Knopf mit einer leeren Hand Shift-klickst, wird ein Inventar mit einem einzelnen Slot erscheinen. Du kannst Gegenstände in diesen Slot geben, um die Zeit, die der Knopf gedrückt ist, zu verlängern - ein Gegenstand erhöht die Dauer um einen Tick.
 
 tile.openblocks.big_button_wood.name=Großer Holzknopf
 
 tile.openblocks.xp_drain.name=XP-Gitter
-tile.openblocks.xp_drain.description=Platziere ein XP-Gittter über einem OpenBlocks-Tank und stelle dich darauf. Deine XP werden in den Tank entleert, was dann für Blöcke wie den Auto-Zaubertisch oder den Auto-Amboss benutzt werden kann.
+tile.openblocks.xp_drain.description=Platziere ein XP-Gittter über einem OpenBlocks-Tank und stelle dich darauf. Deine Erfahrungspunkte werden in den Tank entleert, was dann für Blöcke wie den Automatischen Zaubertisch oder den Automatischen Amboss benutzt werden kann.
 
 tile.openblocks.vacuum_hopper.name=Vakuumtrichter
-tile.openblocks.vacuum_hopper.description=Der Vakuumtrichter saugt Items und XP-Kugeln um sich herum auf. Du kannst die Tabs auf der rechten Seite des Interfaces verwenden, um auszuwählen, zu welchen Seiten etwas ausgegeben werden soll.
+tile.openblocks.vacuum_hopper.description=Der Vakuumtrichter saugt Gegenstände und Erfahrungs-Kugeln um sich herum auf. Du kannst die Tabs auf der rechten Seite der Oberfläche verwenden, um auszuwählen, zu welchen Seiten etwas ausgegeben werden soll.
 
-tile.openblocks.village_highlighter.name=Dorf-Highlighter
-tile.openblocks.village_highlighter.description=Platziere den Dorf-Highlighter in einem Dorf und gib ihm ein Redstone-Signal. Er wird den Umriss des Dorfes darstellen und eine Box im Eisengolem-Spawnbereich zeichnen. Nützlich, wenn du sichergehen willst, dass Dörfer nicht verschmelzen, oder wenn du eine Eisengolem-Farm baust!
+tile.openblocks.village_highlighter.name=Dorf-Markierer
```

Comment:
That just is a horrible translation. Just don't.

Reply:
Do you have an other idea?

Reply:
No "highlighter", it is not german.

Reply:
Ok